### ISSUE TITLE
add color code from other pypi color libraries.

### ISSUE THREAD

**Issue by rec** [rec/tada#51]
What's the XKCD color survey?

https://pypi.python.org/pypi/colour - https://github.com/vaab/colour
https://pypi.python.org/pypi/colormath - https://github.com/gtaylor/python-colormath
https://github.com/jrosebr1/color_transfer
https://github.com/edaniszewski/colorutils  
(Also: https://github.com/edaniszewski/colorutils/wiki/Comparing-Conversion-Algorithms)
https://github.com/njsmith/colorspacious

**Comment by adammhaile** [comment]
Not sure what you are getting at here? XKCD?

**Comment by rec** [comment]
Oh, someone's color library said, "Names taken from the XKCD color survey" so this was a note to look it up.

And here's the link:  https://blog.xkcd.com/2010/05/03/color-survey-results/

**Comment by rec** [comment]
Edited down:

These three are the final shootout.

https://github.com/gtaylor/python-colormath/tree/master/colormath - has a lot of color models
https://github.com/edaniszewski/colorutils - also has a lot of color models.
https://github.com/njsmith/colorspacious - also has a lot of color models.

* Ed was willing to write a paper proving his numbers were better and seems to have a simpler model.
* NJ has some serious math I don't fully understand even!

I'm leaning towards NJ.  He seems to have the least Python-y things and the most computation (he's also using numpy heavily which shows intelligence.

Since I'm doing all the heavy lifting in C++, all the Python-y stuff is a definite distraction.

**Comment by rec** [comment]
Oh, and I should duplicate also the code in Python's own colorsys, which is pure Python and thus could be dramatically sped up...

**Comment by adammhaile** [comment]
Yes please :) I have my own implementation of that, but it's still pure
Python and only faster because I use bytes instead of float.
